src/main/java/pl/javastart/task/Main.java

```diff
@@ -1,8 +1,33 @@
 package pl.javastart.task;
 
+import java.util.Scanner;
+
 public class Main {
 
+    @SuppressWarnings({"checkstyle:Indentation", "checkstyle:WhitespaceAround"})
     public static void main(String[] args) {
 
+        Scanner reader = new Scanner(System.in);
+        int userInput;
+        int num = 0;
+
+        System.out.println("Podaj liczbę: ");
+
+        while ((userInput = reader.nextInt()) != num) {
```

Comment:
Ten warunek jest błędny. Jeżeli podamy 0, to powinien się wyświetlić komunikat, że liczba jest zbyt mała, nie powinniśmy od razu przerwać pętli